main.go

```diff
@@ -288,7 +356,7 @@ func getServer(network Network, addr *net.TCPAddr) (string, string) {
 	}
 	switch network {
 	case "mainnet":
-		return "electrum.petrkr.net", "s50002"
+		return "e.keff.org", "t50001"
```

Comment:
is this intended? also `t50001` indicates a TCP (non-TLS) conn, right?

Reply:
I was having trouble connecting to `electrum.petrkr.net`. It might have been a temporary thing or an issue on my side?

Reply:
It seems to be working fine for me. Not sure. Maybe we can find a solid list of electrum servers somewhere, or (separately from this PR) have a started list of servers and cycle thru them?